configure.ac

```diff
@@ -232,7 +232,12 @@ AS_IF([test x$enable_python != xno],
               AS_IF([test x${PYTHON3} != xno],
                     [AC_SUBST([PYTHON], [$PYTHON3])])])
        AS_IF([test x${PYTHON} != xno],
-             [AC_MSG_CHECKING([whether $PYTHON as ctypesgen])
+             [AC_MSG_CHECKING([if $PYTHON is version 3.6 or later])
+              AS_IF([! ${PYTHON} -c "import sys; assert(sys.version_info >= (3,6,0))" 2>/dev/null],
+                    [AC_MSG_RESULT([no])
+                     AC_MSG_WARN([python module will not be built])],
+                    [AC_MSG_RESULT([yes])])
+              AC_MSG_CHECKING([whether $PYTHON as ctypesgen])
```

Comment:
We might want to make these two checks nested so that we don't look for `ctypesgen` if we don't have Python >=3.6.

Reply:
See [ff06f45](https://github.com/ledatelescope/bifrost/pull/216/commits/ff06f4539f41f9b9d07169fe9b7af2519ffb9cb2).